boxoffice/views/admin_report.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+
+from flask import Response, jsonify
+from .. import app, lastuser
+from coaster.views import load_models, render_with
+from boxoffice.models import db, ItemCollection, Order, Item, LineItem, Assignee, DiscountPolicy, LINE_ITEM_STATUS
+from boxoffice.views.utils import csv_response
+
+
+def jsonify_report(data_dict):
+    return jsonify(org_name=data_dict['item_collection'].organization.name, title=data_dict['item_collection'].title)
```

Comment:
The `title` you're returning in the JSON version is different from the HTML version?